plugin/set.go

```diff
@@ -113,7 +113,7 @@ func (i *Set) start(kind, name string) error {
 		err = server.RegisterBuilder(i.Builders[name])
 	case "post-processor":
 		err = server.RegisterPostProcessor(i.PostProcessors[name])
-	case "provisioners":
+	case "provisioner":
```

Comment:
Woops ! I wonder if we should make of these const/shared vars

Reply:
I think that probably makes sense.